dev/devtool/panel.css

```
@@ -1,60 +1,380 @@
 }
-.section h3 {
+.bunext-section-title {
   margin-bottom: 0.5rem;
-  border-bottom: 1px solid #444;
+  border-bottom: 1px solid var(--bunext-border);
   padding-bottom: 4px;
+  background: var(--bunext-bg-alt);
+  color: var(--bunext-text);
+  font-size: 1.05em;
+  font-weight: 500;
+  padding: 6px 10px 4px 10px;
```

Biome (2.1.2)

[error] Unexpected shorthand property padding after padding-bottom



(lint/suspicious/noShorthandPropertyOverrides)